.github/workflows/build.yml

```
@@ -8,7 +8,37 @@ on:
+        filters: |
+          backend: backend/**
+          frontend: frontend/**
+          webcomponent: webcomponent/**
+          eai: eai/**
+    - run: |
+        echo ${{ toJSON(steps.filter.outputs.changes) }}
+
+
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (5 > 2)

(empty-lines)